### PR TITLE
v3.0 edits Apply feedback 

### PR DESCRIPTION
**Pull request by sguequierre**
* start applying feedback to docs

### CODE REVIEW

File: docs/data-ai/_index.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ Common use cases include:
 {{<imgproc src="/platform-overviews/data-and-AI.png" resize="1200x" style="width:800px" class="aligncenter imgzoom" declaredimensions=true alt="ALT">}}
 </p>
```
Comment (sguequierre) [inline docs/data-ai/_index.md:25]:
```suggestion


```

File: docs/data-ai/capture-data/advanced/how-sync-works.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+linkTitle: "How sync works"
+title: "How sync works"
+tags: ["data management", "data", "services"]
+weight: 12
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "TODO"
```
Comment (sguequierre) [inline docs/data-ai/capture-data/advanced/how-sync-works.md:9]: still a draft

File: docs/data-ai/capture-data/capture-other-sources.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+linkTitle: "Capture other data sources"
+title: "Capture and sync other data sources"
+tags: ["data management", "data", "services"]
+weight: 12
+layout: "docs"
+type: "docs"
+platformarea: ["data"]
+description: "TODO"
```
Comment (sguequierre) [inline docs/data-ai/capture-data/capture-other-sources.md:9]: still a draft. if anybody has any better ideas for a name for this besides "capture other sources" I'm all ears

File: docs/data-ai/reference/data/_index.md [hunk 4]
```diff
@@ -25,35 +25,6 @@ toc_hide: true
 The data management service captures data from one or more {{< glossary_tooltip term_id="resource" text="resources" >}} locally, and syncs it to cloud storage when a connection to the cloud is available.
 You can configure which data you want to capture, as well as the capture rate and the sync frequency.
 
-{{< tabs >}}
```
Comment (sguequierre) [inline docs/data-ai/reference/data/_index.md:28]: I was unsure about moving this over to how sync works. It does describe how sync works in addition to capture

File: docs/data-ai/capture-data/advanced/how-sync-works.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+If the internet becomes unavailable or the machine needs to restart during the sync process, the sync is interrupted.
+If the sync process is interrupted, the service will retry uploading the data at exponentially increasing intervals until the interval in between tries is at one hour, at which point the service retries the sync every hour.
+When the connection is restored and sync resumes, the service continues sync where it left off without duplicating data.
+If the interruption happens mid-file, sync resumes from the beginning of that file.
+
+To avoid syncing files that are still being written to, the data management service only syncs files that haven't been modified in the previous 10 seconds.
+
+## Storage
+
```
Comment (sguequierre) [inline docs/data-ai/capture-data/advanced/how-sync-works.md:61]: does seem to fit into how sync works but I think is also important enough to be in regular reference docs-- thoughts?

Comment (npentrel) [reply docs/data-ai/capture-data/advanced/how-sync-works.md:61]: I suppose this is also a reference page for it. I think the data page in reference right now is not visible, if that stays you can cover the info there too and see how that feels.

The automatic data deletion may need to go somewhere else instead I think, that goes beyond explaining how sync works

File: docs/data-ai/capture-data/capture-other-sources.md (hunk 3 above)
Comment (npentrel) [reply docs/data-ai/capture-data/capture-other-sources.md:9]: sgtm

File: docs/data-ai/reference/data/_index.md (hunk 4 above)
Comment (npentrel) [reply docs/data-ai/reference/data/_index.md:28]: this seems fine to me, you can link to it from here for further reading.

**Pull request merged**